assets/counseling-modal.js

```
@@ -54,6 +139,8 @@ class MyDialog extends HTMLElement {
       btn.addEventListener('click', () => {
         this.pages.forEach((p, i) => p.classList.toggle('active', i === 0));
         this.activePageIndex = 0;
+        this.currentResultIndex = null;
+        this.counselingResult.forEach((result) => result.classList.remove('active'));
```

Biome (2.4.7)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)